ykman-gui/py/yubikey.py

```diff
@@ -274,6 +276,81 @@ def openpgp_get_remaining_pin_retries(self):
                          exc_info=e)
             return None
 
+    def fido_support_ctap(self):
+        try:
+            dev = self._descriptor.open_device(TRANSPORT.U2F)
+            Fido2Controller(dev.driver)
+            return True
+        except Exception as e:
+            logger.debug('Failed to create Fido2Controller', exc_info=e)
+            return False
+
+    def fido_has_pin(self):
+        try:
+            dev = self._descriptor.open_device(TRANSPORT.U2F)
+            controller = Fido2Controller(dev.driver)
+            return controller.has_pin
+        except Exception as e:
+            logger.error('Failed to read if PIN is set', exc_info=e)
```

Comment:
Sure we shouldn't return an error indication here? This will return `undefined` which is likely to be interpreted as a definite `false`.